src/Ion.ts

```diff
@@ -133,4 +134,4 @@ export { SharedSymbolTable } from "./IonSharedSymbolTable";
 export { Timestamp } from "./IonTimestamp";
 export { toBase64 } from "./IonText";
 export { TypeCodes } from "./IonBinary";
-
+export { get_ion_type } from "./IonParserTextRaw";
```

Comment:
I know the current code base uses the C like naming convention but we would like to move to CamelCase. Can we rename the exported function to `getIonTypes` or something to that effect